spec/pb_kits/playbook/props/base_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Playbook
+  module Props
+    describe Base do
+      describe "#validate!" do
+        it "returns true when child class passes validation" do
```

Comment:
You don't want to define classes like this. A better option would have been:
```ruby
truthy_validation_class = Class.new(Playbook::Props::Base) do
  def validate(_value); true; end
end
```
This option avoid defining a constant `TruthyValidationClass`.

Reply:
Cool!